### PR TITLE
fix error popup in Arsenal Garage

### PR DESCRIPTION
**Pull request by commy2**
**When merged this pull request will:**
- fixes error popup `Picture logo_cba_ca.paa not found` when opening the Virtual Garage
- remove a version number from a place that will not get updated
- attempt to hide the invisible target from the Arsenal using `scopeArsenal = 0;`

Predictably `scopeArsenal = 0;` doesn't work. It never works. I just leave it here as a reminder for everyone else that `scopeArsenal` indeed does not work.

### CODE REVIEW

File: addons/main_a3/CfgMods.hpp [hunk 1]
```diff
@@ -2,15 +2,13 @@ class CfgMods {
     class PREFIX {
         author = "$STR_CBA_Author";
         dir = "@CBA_A3";
-        name = "Community Base Addons v3.9.1";
```
Comment (PabstMirror) [inline addons/main_a3/CfgMods.hpp]: change it to 0.0.0
there is code to change on build
https://github.com/CBATeam/CBA_A3/blob/master/tools/make.py#L77
https://github.com/CBATeam/CBA_A3/blob/master/tools/make.py#L597

Comment (commy2) [reply addons/main_a3/CfgMods.hpp]:
```suggestion
        name = "Community Base Addons v0.0.0";
```
Like this?

Comment (PabstMirror) [reply addons/main_a3/CfgMods.hpp]: yeah, also need to change
https://github.com/CBATeam/CBA_A3/blob/master/tools/make.py#L77
versionFiles = ["mod.cpp","addons\main_a3\CfgMods.hpp"]

**Comment by commy2** [comment]
@PabstMirror Merge?

File: addons/main_a3/CfgMods.hpp [hunk 2]
```diff
@@ -2,15 +2,13 @@ class CfgMods {
...
+        name = "Community Base Addons v0.0.0";
         picture = "x\cba\addons\main\logo_cba_ca.paa";
         hidePicture = 1;
         hideName = 1;
         actionName = "Website";
         action = "$STR_CBA_URL";
         description = "Bugtracker: https://github.com/CBATeam/CBA_A3/issues<br/>Documentation: https://github.com/CBATeam/CBA_A3/wiki";
-        logo = "logo_cba_ca.paa";
```
Comment (jonpas) [inline addons/main_a3/CfgMods.hpp]: This works if you have the logo inside a PBO as well, not just root of `@mod` folder.

**Pull request merged**